app.rb

```diff
@@ -0,0 +1,18 @@
+require 'sinatra/base'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+
+  get '/' do
+    @peeps = Peep.all
+    erb:index
+  end
+
+  post '/' do
```

Comment:
Interesting that you're doing this all in the '/' route. I actually really like it from a simplicity perspective. Almost plays like a single-page app. They seem to advocate a more elaborate route structure, which I feel becomes quite cumbersome for lightweight apps. Food for thought.